src/test/java/org/apache/commons/codec/binary/Base64TestData.java

```diff
@@ -30,7 +30,7 @@
  */
 public class Base64TestData {
 
-    public static final String CODEC_101_MULTIPLE_OF_3 = "123";
+    public static final String CODEC_101_MULTIPLE_OF_3 = "124";
```

Comment:
ERhm... what's going on here?

Reply:
I can't say that I remember changing this and I'm not sure why I would have. It clearly doesn't make any sense.

Reply:
I dug in and found the reason: https://issues.apache.org/jira/browse/CODEC-134?focusedCommentId=13222167&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13222167 
It should really be renamed to "three digit valid base64 value" or someting.

Reply:
Ah! Thank you for digging this up! I knew there was some reason behind it, but since I made this change about a year and a half ago, I didn't remember why. Should've changed the constant name while I was at it to make it more clear and possibly add a comment to explain why 124 fits the bill.

Reply:
Note that this has already been merged for `1.13`. The functionality was incomplete and was fixed in CODEC-270 for release `1.14`. We have just released `1.15` that makes this functionality opt-in rather than always on. For version `1.15` trailing bit checking is off by default thus matching the behaviour from `1.12` and earlier where invalid trailing bits are discarded. You can enable it for strict decoding where invalid bits raise an exception when decoding.

Reply:
Yes, I found the strangely named variable when going through the changes in 1.15 (because 1.14 broke our builds). It just struck me that 123 is indeed a multiple of 3, but 124 isn't. Later on, I found out that it probably was meant as "valid b64 of length 3". Which works for 124 but not 123. So it's a poorly named variable.

Reply:
This variable dates back to an old bug for CODEC-101. The variable should have a byte length that is a multiple of 3. The issue is that the string `123` is not a valid base 64 encoding, so it had to change for this PR. I suggest a name change to `CODEC_101_INPUT_LENGTH_IS_MULTIPLE_OF_3`.